pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -1662,10 +1662,15 @@ class Timedelta(_Timedelta):
 
         # GH 30543 if pd.Timedelta already passed, return it
         # check that only value is passed
-        if isinstance(value, _Timedelta) and unit is None and len(kwargs) == 0:
+        if isinstance(value, _Timedelta):
+            # 'unit' is benign in this case, but e.g. days or seconds
+            #  doesn't make sense here.
+            if len(kwargs):
+                # GH#48898
+                raise ValueError(
+                    "Cannot pass both a Timedelta input and timedelta keyword arguments."
```

Comment:
Might be nice to show what `kwags` were passed

Reply:
good idea, updated

Reply:
updated+green